src/custom/hooks/useERC20Permit/useERC20PermitMod.ts

```diff
@@ -120,15 +120,24 @@ export * from '@src/hooks/useERC20Permit'
 //   { name: 'allowed', type: 'bool' },
 // ]
 
-// export function useERC20Permit(
-//   currencyAmount: CurrencyAmount<Currency> | null | undefined,
-//   spender: string | null | undefined,
-//   overridePermitInfo: PermitInfo | undefined | null
-// ): {
-//   signatureData: SignatureData | null
-//   state: UseERC20PermitState
-//   gatherPermitSignature: null | (() => Promise<void>)
-// } {
+/* eslint-disable @typescript-eslint/no-unused-vars */
+export function useERC20Permit(
+  currencyAmount: CurrencyAmount<Currency> | null | undefined,
+  spender: string | null | undefined,
+  overridePermitInfo: PermitInfo | undefined | null
+): {
+  signatureData: SignatureData | null
+  state: UseERC20PermitState
+  gatherPermitSignature: null | (() => Promise<void>)
+} {
+  // Mod: make useERC20Permit a noop. Permissable ERC20 currently not supported by our contracts
+  return {
+    signatureData: null,
+    state: UseERC20PermitState.NOT_APPLICABLE,
+    gatherPermitSignature: null,
+  }
+}
+/* eslint-enable @typescript-eslint/no-unused-vars */
```

Comment:
Are these TS disabling comments necessary?

Reply:
Yes, because I'm not using any of the input paramenters.
TS complains and build fails otherwise.